datatypes/src/util/identifiers.rs

```diff
@@ -4,6 +4,7 @@ pub trait Identifier: Sized {
 }
 
 #[macro_export]
+#[allow(clippy::crate_in_macro_def)]
```

Comment:
```suggestion
```
Why don't call it `$crate` like at the other places?

Reply:
because it actually is `crate` and not `$crate` 🙈 maybe a bad smell but I don't know how to do it differently

Reply:
[f3750c1](https://github.com/geo-engine/geoengine/pull/524/commits/f3750c15db460ba30bf9831ede6d603cf6990158)